doc/references.bib

```diff
@@ -616,6 +616,16 @@ @article{HaufeEtAl2014
   year = {2014}
 }
 
+@article{HaufeEtAl2014,
+	author = {Haufe, Stefan and D{\"a}hne, Sven and Nikulin, Vadim V},
+	doi = {https://doi.org/10.1016/j.neuroimage.2014.06.073},
+	journal = {NeuroImage},
+	pages = {583-597},
+	title = {Dimensionality reduction for the analysis of brain oscillations},
+	volume = {101},
+	year = {2014}
+}
```

Comment:
CircleCI treats warnings as errors so you [go to the log](https://app.circleci.com/pipelines/github/mne-tools/mne-python/4560/workflows/249be028-5b99-43db-ba38-ee6efd8ce12a/jobs/23294) and search "warning" you will find errors usually, but this one doesn't even require that:
```
pybtex.database.BibliographyDataError: repeated bibliograhpy entry: HaufeEtAl2014
```

Reply:
alright, I will check this double bibliography also.. thanks!